appveyor.yml

```diff
@@ -48,14 +52,14 @@ install:
     - python --version
     - python -c "import struct; print(struct.calcsize('P') * 8)"
 
-    # Install the mingw compiler, but only if we're on Python 2.7
-    - pip install --no-cache-dir --pre --index https://pypi.anaconda.org/carlkl/simple "mingwpy; python_version == '2.7'"
+    # Install the mingw compiler, but only if we're on Python 2.7 or 3.4
+    - pip install --no-cache-dir --pre --index https://pypi.anaconda.org/carlkl/simple "mingwpy; python_version < '3.5'"
 
 build_script:
     # Install the dependencies of the project.
     - python -m pip install %BUILD_DEP%
     # Build wheel
-    - python -m pip wheel --wheel-dir=wheelhouse --no-binary=%PACKAGE_NAME% %PROJECT_SPEC%
+    - python -m pip wheel --wheel-dir=wheelhouse --no-binary=%PACKAGE_NAME% %PROJECT_SPEC% %EXTRA_BUILD_OPTIONS%
```

Comment:
I don't think `pip wheel` can take a `--compiler` option. `bdist_wheel` does for sure.
I think pip wheel has a `--build-option <options>` that takes a string of bdist_wheel options.
You may well just call bdist_wheel directly. I don't see much gain from calling pip wheel here